app/lambda/get_transcription_comprehend/lambda_function.py

```diff
@@ -0,0 +1,71 @@
+import json
+import sys
+
+sys.path.append("../")
+import boto3
+from lambda_base import LambdaBase
+from constant_variables import *
+
+class GetTranscriptionComprehendLambda(LambdaBase):
+    def __init__(self): 
+        pass
+
+    def putItem(self, PatientId, HealthCareProfessionalId, SessionName, SessionId, TimeStampStart, TimeStampEnd, TranscribeS3Path, ComprehendS3Path):
+        info = {DATASTORE_COLUMN_SESSION_ID : SessionId,
+                DATASTORE_COLUMN_PATIENT_ID: PatientId,
+                DATASTORE_COLUMN_HEALTH_CARE_PROFESSSIONAL_ID: HealthCareProfessionalId,
+                DATASTORE_COLUMN_SESSION_NAME: SessionName,
+                DATASTORE_COLUMN_COMPREHEND_S3_PATH: ComprehendS3Path,
+                DATASTORE_COLUMN_TRANSCRIBE_S3_PATH: TranscribeS3Path,
+                DATASTORE_COLUMN_TIMESTAMP_START: TimeStampStart,
+                DATASTORE_COLUMN_TIMESTAMP_END: TimeStampEnd}
+        Session().createSession(info)
+        return SessionId
+
+    def getKeyName(self, sessionId, category, fileType):
+        return 'public/'+category+'-medical-output/'+sessionId+'/'+sessionId+'-session-'+category+'.'+fileType
+
+    def handle(self, event, context):
+        try:
+            print(event)
+            sessionId = event['queryStringParameters']['sessionId'] if 'sessionId' in event['queryStringParameters'] else None
+            print(sessionId)
+            bucket = 'mtastack-mtastackstorages3bucketc161f3b3-1tfncqzctldb1'
+            comprehend_key = self.getKeyName(sessionId,'comprehend','json')
+            transcribe_key = self.getKeyName(sessionId,'transcribe','txt')
+            print(comprehend_key)
+            print(transcribe_key)
+            client = boto3.client('s3', region_name='us-west-2')
```

Comment:
You dont need to harcode the region in your Lambda.
AWS Lambda supports some default run time variable. Have a look at : https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html

You can access the variable via os.environ['AWS_REGION']

Reply:
Done.